operator/src/main/resources/application.properties

```diff
@@ -129,50 +136,16 @@ managedkafka.zookeeper.volume-size=10Gi
 managedkafka.kafka.max-connections=3000
```

Comment:
IMO we should totally remove any properties represented by KafkaInstaceConfiguration in default `application.properties`. It may get confusing what default, who overridden what etc.

Reply:
Without additional refactoring some have to remain as they aren't actually part of the KafkaInstanceConfiguration.  I'm fine with leaving the rest of the the defaults in the main application.properties.

Reply:
Several of the canary ones at least.

Reply:
If are saying these
```
    @ConfigProperty(name = "managedkafka.canary.producer-latency-buckets")
    @ConfigProperty(name = "managedkafka.canary.endtoend-latency-buckets")
    @ConfigProperty(name = "managedkafka.canary.connection-check-latency-buckets")
    @ConfigProperty(name = "managedkafka.canary.status-time-window-ms")
    @ConfigProperty(name = "managedkafka.canary.init-timeout-seconds")
    @ConfigProperty(name = "managedkafka.canary.init-enabled")
```
seems like they need to be fixed to be consistent imo.

Reply:
My point is that it's optional work - not tied to what is being asked for here.  Only if you think those properties will get changed per instance type do we to have them at this time on the kafkainstanceconfiguration.

Reply:
agree, they are not related. I was looking from the point of view to keep it consistent with other properties. the way these properties are defined the author seemed to think they can change at the instance level but did not take the time to do it consistently with others IMO. we can come back in another JIRA and fix them then.